static/js/message_list.js

```diff
@@ -35,9 +35,19 @@ export class MessageList {
         this.narrowed = this.table_name === "zfilt";
         this.num_appends = 0;
 
+        this.resume_reading();
+
```

Comment:
Changed this to just initialize `this.reading_prevented` to false, since that'll convert to typescript better.